assets/css/style.css

```diff
@@ -794,20 +794,102 @@ button:disabled {
     padding: 2vw 0;
   }
 
+  .mail-logo {
+    display: none;
+  }
+
+  /* Mobile menu link */
+  .mobile-menu a {
+    color: #fff;
+    text-decoration: none;
+  }
+
   .list-item a:hover,
   button:hover {
     transform: none;
     box-shadow: none;
   }
 
+  .hamburger-icon .bar {
+    height: 0.6vw;
+    width: 6vw;
+  }
+
+  /* Mobile menu */
+  .mobile-menu {
+    display: none;
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    background-color: #3c3a39;
+    color: #fff;
+    z-index: 999;
+  }
+
+  /* Show mobile menu when active */
+  .mobile-menu.active {
+    display: flex;
+  }
```

Comment:
- [ ] Great work on incorporating the mobile menu into your portfolio! :+1: However, I suggest that you consider preventing users from accessing other sections of your portfolio while the mobile menu is open. This can be achieved by using the `position: fixed` property. By implementing this, users can focus solely on the menu and have a better browsing experience on mobile devices.
- [ ] Although your efforts are commendable, there seems to be an issue with the hamburger button not disappearing when users switch to the desktop page as you can see in the video below:


https://user-images.githubusercontent.com/105117832/223283333-f803b87a-1720-42f6-aa45-63ef5211ebef.mp4

Reply:
Thank yoy very much for your amazing feedback.
I solver those problems.
Have a good day.